assets/js/timetree.js

```diff
@@ -554,6 +628,19 @@ function TreeGraph({ nodes, links, centuries }) {
     return 0;
   }
 
+  // since svg width depends on mobile/desktop, adjust on resize
+  window.onresize = () => {
+    let newWidth = svgWidth();
+    // if the width has changed, update width, min x, svg viewbox,
+    // and reposition the simulation
+    if (newWidth != width) {
+      width = newWidth;
+      min_x = -width / 2;
+      svg.attr("viewBox", [min_x, min_y, width, height]);
+      simulation.tick(); // should reflow and update everything with new sizes...
```

Comment:
(Larger, indirect thought, but) I wonder if a lot of this code here could be abstracted/generalized once Lenape is done to be a template for other d3+hugo focused projects. A lot of the code seems extensible in that sense